_includes/build/openapi.html

```diff
@@ -0,0 +1,29 @@
+<h2>
+	Generating Client Code
+</h2>
+
+	<h3>
+		OpenAPI Code Generator
+	</h3>
+
+		<p>
+			An OpenAPI v3 JSON schema file, which represents the service capabilities of the API, can be accessed through BCDA’s Swagger page.This file can be used to generate client code to interact with BCDA.
```

Comment:
@Dave-Sukharan - can we link to "BCDA's Swagger page" and "Swagger Codegen" from this include?

Reply:
We can link out to both versions:

v1 - https://sandbox.bcda.cms.gov/api/v1/swagger/#/
v2 - https://sandbox.bcda.cms.gov/api/v2/swagger/#/